test/e2e_flags.go

```diff
@@ -43,17 +42,18 @@ const (
 	BrokerNameUsage = "When testing a pre-existing broker, specify the Broker name so the conformance tests " +
 		"won't create their own."
 	BrokerNamespaceUsage = "When testing a pre-existing broker, this variable specifies the namespace the broker can be found in."
+	BrokerClass          = "MTChannelBasedBroker"
```

Comment:
Why leaving `BrokerClass` configurable?

Reply:
Because the only possible value is MTChannelBasedBroker. See the line which I removed: https://github.com/knative/eventing/pull/6966/files#diff-1bee0234a4a7bc66483f6aaffb29c6b0d6fc372bde980d3023c564fb30219893L77

Not sure if we'll have more options in the future. I can return it back but it's just redundant code right now.

Reply:
/lgtm
/approve